local-cli/rnpm/wpf/package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "rnpm-plugin-wpf",
-  "version": "0.1.3",
+  "version": "0.2.9",
   "description": "rnpm plugin that generates a Windows template project with WPF",
```

Comment:
Can we just trash this plugin from the repo? I don't think it's in use. That or maybe we can actually split the two plugins...

Reply:
I don't remember why this was merged with the `rnpm-plugin-windows` plugin in the first place (any input @matthargett, @pre10der89, or @kevinvangelder?)

---
In reply to: [145518629](https://github.com/Microsoft/react-native-windows/pull/1457#discussion_r145518629) [](ancestors = 145518629)

Reply:
I simplified it with this refactor to basically point to the rnpm-plugin-windows repo. I don't think we should trash it because it has been out for a while and people are downloading it (see the stats): https://www.npmjs.com/package/rnpm-plugin-wpf

At least with this refactor, you'll never need to update it again.